NETWORK/NetworkAddEntityToSynchronisedScene.md

```diff
@@ -5,16 +5,42 @@ ns: NETWORK
 
 ```c
 // 0xF2404D68CBC855FA 0x10DD636C
-void NETWORK_ADD_ENTITY_TO_SYNCHRONISED_SCENE(Entity entity, int netScene, char* animDict, char* animName, float speed, float speedMulitiplier, int flag);
+void NETWORK_ADD_ENTITY_TO_SYNCHRONISED_SCENE(Entity entity, int netScene, char* animDict, char* animName, float blendInSpeed, float blendOutSpeed, float flag);
 ```
 
 
-## Parameters
-* **entity**: 
-* **netScene**: 
-* **animDict**: 
-* **animName**: 
-* **speed**: 
-* **speedMulitiplier**: 
-* **flag**: 
+### Parameters
```

Comment:
Things such as `## Parameters` and `## Example` don't need triple hashes, only custom headers do. Custom headers in this case are `### Animation Dictionaries`, `### Animations`, `### Used With`. Basically any headers _that you add that aren't commonly used in native docs._